packages/to-dts/lib/traverse.js

```diff
@@ -74,7 +74,15 @@ function traverseFn(g) {
       }
 
       if (def.entries) {
-        arr.push(...g.traverse(def.entries, { parent: tsType, path: `${p}/entries`, flags: 0 }));
+        if (def.kind === 'function') {
```

Comment:
Add a namespace to collect properties when the definition is a function and has entries. The namespace uses the name and the function in order for typescript to merge the entities.